web/src/beta/components/fields/SelectField/index.tsx

```diff
@@ -22,6 +22,7 @@ export type Props = {
   // Property field
   name?: string;
   description?: string;
+  className?: string;
```

Comment:
Since you aren't using the `timezone` className is this change still necessary? Can you clean up any parts related to that that aren't needed? 🙏🏻

Reply:
@KaWaite  I come to realize that className is used to override the `PickerWrapper` height and add scroll, I removed this but the select in timezone look ugly, if you may have any suggestion to add the height, overflow and with in this component.